planning/planning_test_utils/README.md

```diff
@@ -1,92 +1,38 @@
-# Autoware Planning Test Manager
+# Planning Test Utils
 
 ## Background
 
-In each node of the planning module, when exceptional input, such as unusual routes or significantly deviated ego-position, is given, the node may not be prepared for such input and could crash. As a result, debugging node crashes can be time-consuming. For example, if an empty trajectory is given as input and it was not anticipated during implementation, the node might crash due to the unaddressed exceptional input when changes are merged, during scenario testing or while the system is running on an actual vehicle.
+Several planning components and modules have already adopted unit testing, so a common library to ease the process of writing unit tests is necessary.
 
 ## Purpose
 
-The purpose is to provide a utility for implementing tests to ensure that node operates correctly when receiving exceptional input. By utilizing this utility and implementing tests for exceptional input, the purpose is to reduce bugs that are only discovered when actually running the system, by requiring measures for exceptional input before merging PRs.
+The objective of the `planning_test_utils` is to develop a unit testing library for the planning components. This library will include
```

Comment:
```suggestion
The objective of the `planning_test_utils` is to develop a unit testing library primarily designed for the planning components. *However*, this library can also be utilized for testing other components as well. This will include:
```

Reply:
`primarily designed` is redundant. It doesn't add any additional useful information towards the sentence.

`can also be utilized for testing other components as well. This`
Redundant information as well, since we will be moving this to the common folder later. (the name planning test utils will be changed as well to only utils)

Reply:
I also remove the planning part so that once the document is moved to `common`, we dont need to fix the document anymore.